src/util/http/websocket/QueryHub.cpp

```diff
@@ -38,7 +38,7 @@ QueryHub::createOrAcquireDistributorInternalUnsafe(QueryId queryId) {
                    socketDistributors = socketDistributors_, queryId]() {
         auto future = net::dispatch(net::bind_executor(
             globalStrand,
-            std::packaged_task<void()>([&socketDistributors, &queryId]() {
+            std::packaged_task<void()>([socketDistributors, queryId]() {
```

Comment:
I think you can even deduce the signature:
```suggestion
            std::packaged_task{[socketDistributors, queryId]() {...}};
```